.github/workflows/checks.yml

```diff
@@ -1,35 +1,41 @@
 name: Checks
```

Comment:
@usamaidrsk will you rename this workflow to meaningful title please. `Checks` doesn't give idea to contributors what exactly the check is about.